ext/opcache/jit/zend_jit_arm64.dasc

```diff
@@ -11839,6 +11838,21 @@ static int zend_jit_fetch_obj(dasm_State          **Dst,
 			|	cbnz FCARG2x, >1
 			|.cold_code
 			|1:
+			|	ldr TMP1w, [FCARG2x, #offsetof(zend_property_info, flags)]
+			|	tst TMP1w, #ZEND_ACC_READONLY
+			|	bne >3
+			|	IF_NOT_TYPE REG2w, IS_OBJECT_EX, >2
```

Comment:
Why is this IS_OBJECT_EX in the arm64 code for IF_NOT_TYPE? (x86 code is using IS_OBJECT and has passing tests)

The SET_ZVAL_TYPE_INFO use looks correct, I'm just confused about the IF_NOT_TYPE use

Reply:
Objects should only occur with IS_OBJECT_EX type info. Note that this code loads the whole type_info, while x86 loads only type.